RELEASE-NOTES.txt

```diff
@@ -11,6 +11,7 @@
 ### Fixes
 
  - Rework WordPress.com signin to prevent infinite looping and login failures [#1627](https://github.com/Automattic/simplenote-electron/pull/1627)
+ - Fixed link to release notes in updater config
```

Comment:
It's fine as is but it could be helpful to indicate what was wrong and what is right

```
 - Update link to release-notes in updater config: CHANGELOG -> RELEASE_NOTES
```